apps/judicial-system/web/src/routes/Court/Indictments/Conclusion/Conclusion.tsx

```diff
@@ -232,14 +238,21 @@ const Conclusion: FC = () => {
                   file.status === 'done',
               )
             )
-          case CaseIndictmentRulingDecision.FINE:
           case CaseIndictmentRulingDecision.CANCELLATION:
-          case CaseIndictmentRulingDecision.MERGE:
+          case CaseIndictmentRulingDecision.FINE:
             return uploadFiles.some(
               (file) =>
                 file.category === CaseFileCategory.COURT_RECORD &&
                 file.status === 'done',
             )
+          case CaseIndictmentRulingDecision.MERGE:
+            return (
+              uploadFiles.some(
+                (file) =>
+                  file.category === CaseFileCategory.COURT_RECORD &&
+                  file.status === 'done',
+              ) && workingCase.mergeCase?.id !== undefined
```

Comment:
Please consider using `!workingCase.mergeCase?.id` here as it also handles `null`.

Reply:
Good point! Will make the change 😊